dotcom-rendering/src/web/components/Card/components/HeadlineWrapper.tsx

```diff
@@ -1,17 +1,46 @@
 import { css } from '@emotion/react';
+import { from } from '@guardian/source-foundations';
 
 type Props = {
 	children: React.ReactNode;
+	imagePosition?: ImagePositionType;
+	imagePositionOnMobile?: ImagePositionType;
+	imageUrl?: string;
+	starRating?: number;
 };
 
-export const HeadlineWrapper = ({ children }: Props) => (
+export const HeadlineWrapper = ({
+	children,
+	imagePosition,
+	imagePositionOnMobile,
+	imageUrl,
+	starRating,
+}: Props) => (
 	<div
 		css={css`
 			padding-bottom: 8px;
 			padding-left: 5px;
 			padding-right: 5px;
-			padding-top: 1px;
+			padding-top: ${imageUrl &&
+			imagePositionOnMobile &&
+			imagePositionOnMobile === 'left' &&
+			starRating
+				? '0px'
+				: '1px'};
```

Comment:
I think this will have the unintentional bug that if the starRating is `0` then we won't get the padding (this does happen, though I can't remember the examples - The Guardian can be harsh on their ratings!). We probably want to do the check `starRating !== undefined`. We could do this here or change HeadlineWrapper to take the prop `hasStarRating` instead, and simplify the logic to just the use of the component in Card?

Reply:
Here’s the full list: https://content.guardianapis.com/search?star-rating=0&api-key=test

Reply:
Harsh!

Reply:
@OllysCoding good point. Pushed the change. I'm also glad this line instigated the introduction of a new ESLint rule in our codebase!